### PR TITLE
update flags and docs, address #59

### PR DESCRIPTION
**Pull request by lziest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,8 +111,17 @@ following options are required and take a path to these files. These two
...
 presented by the server when a client connects.
 
-     --server-cert 
-     --server-key
+     -cert
+     -key
+
+In order to get server credential, we can use the following parameter to
+generate the server private key and certificate signing request (CSR). It
+will create a private key stored `server-key.pem` (or the path specified
```
Comment (kisom) [inline README.md]: Probably should be "store as `server-key.pem`".

File: cmd/gokeyless/gokeyless.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ func init() {
 	flag.StringVar(&port, "port", "2407", "Keyless port on which to listen")
 	flag.StringVar(&metricsAddr, "metrics-addr", "localhost:2406", "address where the metrics API is served")
 	flag.StringVar(&pidFile, "pid-file", "", "File to store PID of running server")
-	flag.BoolVar(&manualMode, "manual-activation", false, "Manually activate the keyserver by writing the CSR to stderr")
+	flag.BoolVar(&manualMode, "manual-activation", false, "The keyserver generates key and CSR files and exits. Use the CSR to get server certificate issued manually.")
```
Comment (kisom) [inline cmd/gokeyless/gokeyless.go]: Maybe "generates 'key and CSR, and exits.'".

File: README.md (hunk 1 above)
Comment (lziest) [inline README.md]: fixed.

File: cmd/gokeyless/gokeyless.go (hunk 2 above)
Comment (lziest) [inline cmd/gokeyless/gokeyless.go]: fixed.

**Pull request merged**